### DISCUSSION TITLE
How can I run validate on a specific subset of files?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/validate#993] [Q&A] [answered]
Is there a way to ONLY run the small subset of files that threw the Integrity errors? It only affected 194/2430 files. I have not found the right combination of options to use when running Validate to retest a specific selection.

**Comment by pds-ops** [comment, marked as answer]
There are a couple ways you can do it, but the easiest is to create a target manifest with paths to each of the files.

Manifest:
```
/path/to/product/x.xml
/path/to/product/y.xml
/path/to/product/z.xml
```

Then run validate with the `target-manifest` flag:
```
validate --target-manifest my-manifest.txt
```

**Comment by thareUSGS** [comment]
I see their issue now. This gets complicated for Validate quickly. 

So, to be able to find their remaining _warning.integrity.reference_not_found_ issues, they can't just test the files independently. They have to do a "validate -R pds4.bundle" run. This type of run doesn't limit itself to the "--target.manifest" file listing. For example, running the command below will still run over all files.

`validate -R pds4.bundle bundle.xml --target-manifest test.list`

Sounds tricky to implement, but any thoughts?
